### PR TITLE
Removes dependency on 'prologic/remark-lint:latest' and fix #44

### LINKED ISSUE #44: Error: Cannot parse file `.remarkrc.yaml`
I'm pretty sure it's not `action-remark-lint` throwing this error, but it's strange because running `remark --rc-path .remarkrc.yaml .` locally does not yield any errors. However, on GitHub Actions, `action-remark-lint` throws [the following error](https://github.com/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pull/330/checks?check_run_id=3415924654):

> ```
> Error: Cannot parse file `.remarkrc.yaml`
> ```

Any ideas what may be causing this?

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request removes the dependency on the 'prologic/remark-lint:latest' dockerfile base. This was done so that we can easier debug the action while also keeping all the packages up to date. This commit also adds a '.remarkrc' file to the `testdata` folder. It also fixes #44.

### Tests
- ✔️ Passed: https://github.com/rickstaa/action-test-repo/pull/61

### CODE REVIEW

**Comment by rickstaa** [comment]
@asbjornu I went ahead and created a new `DockerFile`.  By doing this, #44 also seems to be fixed. I kept the code close to our original code since I am a bit short on time. In the future, we might want to transform it into bourne shell format (i.e., `sh`) to make the image a bit smaller. 

Let me know if there are any points you would like to improve or feel free to merge the pull request.

**Comment by rickstaa** [comment]
@asbjornu I now attached the `bump:major` tag to the pull request since it is a big change. Feel free to replace this tag with the `bump:minor` if you think that reflects the size of the change better.

**Comment by rickstaa** [comment]
Moment, I will fix the suggestions made by the hado-lint action.

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,13 @@
-FROM prologic/remark-lint:latest
+FROM node:current-alpine
 
 ENV REVIEWDOG_VERSION=v0.13.0
 
-RUN apk --no-cache add bash~=5.0
+RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b /usr/local/bin/ ${REVIEWDOG_VERSION}
```
[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile:5]: Will be fixed in next commit.

File: Dockerfile [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
+RUN apk --no-cache add git bash
```
[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile:6]: Fixed in next commit.

File: Dockerfile [hunk 3]
```diff
@@ -1,15 +1,13 @@
...
 
-SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b /usr/local/bin/ ${REVIEWDOG_VERSION}
-
-RUN apk --no-cache -U add git
+RUN npm install -g remark-cli remark-preset-lint-recommended remark-lint
```
[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile:9]: @asbjornu I think it is nicer to keep these command separated as they do different things.

[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile]: @asbjornu I would the action to use the latest remark lint version. What do you think?

File: Dockerfile [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
-RUN apk --no-cache add bash~=5.0
+RUN apk --no-cache add git bash~=5.1
 
+ENV REVIEWDOG_VERSION=v0.13.0
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh | sh -s -- -b /usr/local/bin/ ${REVIEWDOG_VERSION}
+RUN wget -O - -q https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh| sh -s -- -b /usr/local/bin/ ${REVIEWDOG_VERSION}
 
-RUN apk --no-cache -U add git
+RUN npm install -g remark-cli@latest remark-preset-lint-recommended@latest remark-lint@latest
```
[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile:9]: @asbjornu I think it is nicer to keep these command separated as they do different things.

File: Dockerfile [hunk 5]
```diff
@@ -1,15 +1,14 @@
-FROM prologic/remark-lint:latest
+FROM node:current-alpine
 
-ENV REVIEWDOG_VERSION=v0.13.0
-
-RUN apk --no-cache add bash~=5.0
+RUN apk --no-cache add git bash~=5.1
```
[the remark this answers is not in the document]
Comment (asbjornu) [reply Dockerfile]: Any reason to not pin the version of `git` here?

File: Dockerfile (hunk 4 above)
[the remark this answers is not in the document]
Comment (asbjornu) [reply Dockerfile:9]: Agreed

**Comment by rickstaa** [comment]
@asbjornu New tests also pass.

https://github.com/rickstaa/action-test-repo/pull/62

File: Dockerfile (hunk 3 above)
[the remark this answers is not in the document]
Comment (asbjornu) [reply Dockerfile]: I would prefer to pin the versions, but allow for them to be automatically updated by a GitHub Action (or Dependabot or Renovate or whatever supports it). I've created a [custom JSON based scheme for APT](https://github.com/SwedbankPay/jekyll-plantuml-docker/blob/main/.github/workflows/apt-update.yml).

[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile]: @asbjornu I think your solution is better. Do you have the time this week to implement that change?

[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile]: @asbjornu I think your solution is better. Do you have the time to implement that change?

[the remark this answers is not in the document]
Comment (asbjornu) [reply Dockerfile]: Yes, but they should not be breaking so we can do them later. Can you please pin to a specific version in this PR and I'll follow up with a JSON-based solution?

File: Dockerfile (hunk 4 above)
[the remark this answers is not in the document]
Comment (asbjornu) [reply Dockerfile:9]: Please pin to specific versions and I'll create an auto-update scheme in a future PR.

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by rickstaa** [comment]
Latest test results:
- ✔️ Passed: https://github.com/rickstaa/action-test-repo/pull/63#pullrequestreview-744705806

File: Dockerfile (hunk 5 above)
[the remark this answers is not in the document]
Comment (rickstaa) [reply Dockerfile]: @asbjornu No, I'm happy to pin git aswell.

**Review by asbjornu — APPROVED** [review]

**Comment by asbjornu** [comment]
Awesome!

**Pull request merged**